spec/dor_config.rb

```diff
@@ -22,11 +22,7 @@
     catalog.url 'http://example.edu/catalog/mods'
   end
 
-  gsearch do
-    rest_url 'https://example.edu/gsearch/rest'
-    url      'https://example.edu/solr/gsearch'
-  end
-  solrizer.url     'https://example.edu/solr/solrizer'
+  solr.url     'https://example.edu/solr/solrizer'
```

Comment:
I noticed there are existing tests that expect `https://example.edu/solr/solrizer`, and it wouldn't surprise me if there's other HTTP mocking too. I think it's a great idea, but I'd certainly prefer to defer it to a different pull request.